docs/developers.rst

```diff
@@ -0,0 +1,56 @@
+For Developers
+==============
+
+The Radio Astronomy Software Group welcomes contributions from anyone. Full contribution guidelines and code of conduct may be found on the `RASG Github Pages <https://radioastronomysoftwaregroup.github.io>`_.
+
+This page discusses more specific points to help developers contribute to ``pyuvsim``.
+
+Computing Resources
+------------------
+
+Instrument simulation is a computationally-demanding task, and ``pyuvsim`` is designed to run a brute-force calculation that requires much computing power. There is a continual effort to improve in performance, but for the foreseeable future the best results can be obtained using large, distributed computing clusters.
+
+Much of the development and testing of ``pyuvsim`` was done on the Oscar computing cluster at Brown University's Center for Computation and Visualization (CCV_). This cluster is managed with the Simple Linux Utility Resource Manager (SLURM). Though ``pyuvsim`` can be run on most Linux or Mac-OS systems, its operation with large numbers of times/baselines/frequencies/sources or especially complicated beam objects has not been tested well. Testing within other cluster management environments is encouraged.
+
+.. _CCV: https://docs.ccv.brown.edu/oscar/
+
+
+Running and Comparing Reference Simulations
+-------------------------------------------
+
+The *reference simulations* comprise a set of standard simulations with well-defined, simple parameters that most instrument simulation tools should be capable of running. As such, they represent a standard point of comparison between ``pyuvsim`` and other simulators. They are also used to track that ``pyuvsim`` produces consistent results over time.
+
+For some types of pull requests, we ask that the reference simulations be re-run with the new code, and the resulting files be compared to an older set, to confirm that either (1) results or (2) that any changes are understood. The older set of files are stored in a Google drive, and are downloadable using scripts in the ``reference_simulations`` directory. If a change to the reference simulations is expected, it should be documented in the CHANGELOG and the new simulation file given to the project managers to store on the Google drive.
```

Comment:
This part: `to confirm that either (1) results or (2) that any changes are understood.` seems to be missing a word or two.

Reply:
Ah, thanks for catching that. It should be fixed now.